gcp-data-drive/fs.go

```diff
@@ -58,7 +57,7 @@ func (f *fsDataPlatform) getData(ctx context.Context) ([]byte, error) {
 		return nil, err
 	}
 
-	// Create a map to hold our firestore result set.
+	// Create a slice of map string interface to hold our firestore result set.
```

Comment:
```suggestion
	// Create a slice to hold our Firestore results.
```

Reply:
Updated to "slice of maps" to be consistent with the BQ comments.